src/_pytest/config/__init__.py

```diff
@@ -761,6 +761,18 @@ def _initini(self, args):
         self._parser.extra_info["inifile"] = self.inifile
         self._parser.addini("addopts", "extra command line options", "args")
         self._parser.addini("minversion", "minimally required pytest version")
+        self._parser.addini(
+            "python_files",
```

Comment:
Not a big fan of having this options here, as they are clearly related to the `python` plugin.

If the cause is because of assertion rewrite (https://github.com/pytest-dev/pytest/issues/5277#issuecomment-493232024), can we just use a default there (using an empty `[]` if not defined)? Would that allow `-p no:python` to work?

Reply:
Done, using the actual default.
And moved `usefixtures` to the fixtures plugin.

Reply:
Please re-review.